content/en/docs/getting-started/integration/policy-controllers/operator-policy.md

```
@@ -0,0 +1,292 @@
+   spec:
+   predicates:
+      - requiredClusterSelector:
+            celSelector:
+            celExpressions:
+               - managedCluster.metadata.name == "cluster1"
+   ```
+
+3. Apply the policy to the hub cluster:
+
+   ```Shell
+   $ kubectl apply -n default -f policy-eso.yaml --context ${CTX_HUB_CLUSTER}
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)